examples/picking/main.js

```diff
@@ -1,28 +1,38 @@
-import { configureViewer, loadDefaultModel } from '../helpers.js';
+import { loadDefaultStreamKey } from '../helpers.js';
+import { ColorMaterial } from 'https://unpkg.com/@vertexvis/viewer@latest/dist/viewer/index.esm.js';
 
 document.addEventListener('DOMContentLoaded', () => {
   main();
 });
 
 async function main() {
   const viewer = document.querySelector('vertex-viewer');
-  await configureViewer(viewer);
-  await loadDefaultModel(viewer);
+  await loadDefaultStreamKey(viewer);
 
   viewer.addEventListener('tap', async (event) => {
     const { position } = event.detail;
     const scene = await viewer.scene();
     const raycaster = await scene.raycaster();
 
-    const { bomItems } = await raycaster.intersectItems(position).execute();
+    const result = await raycaster.hitItems(position);
+    console.log(result);
```

Comment:
I'd say either remove this log, or add a prefix to it: `console.log('hit results', result)`

Reply:
Yeah, I think this was left over from some testing I was doing. I'll remove